fastlane/lanes/localization.rb

```diff
@@ -162,24 +162,25 @@
   desc 'Updates the AppStoreStrings.po file for the WordPress app with the latest data'
   lane :update_wordpress_appstore_strings do |options|
     source_metadata_folder = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'metadata', 'default')
+    custom_metadata_folder = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'appstoreres', 'metadata', 'source')
 
     files = {
       whats_new: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt'),
       app_store_name: File.join(source_metadata_folder, 'name.txt'),
       app_store_subtitle: File.join(source_metadata_folder, 'subtitle.txt'),
       app_store_desc: File.join(source_metadata_folder, 'description.txt'),
       app_store_keywords: File.join(source_metadata_folder, 'keywords.txt'),
-      'standard-whats-new-1' => File.join(source_metadata_folder, 'standard_whats_new_1.txt'),
-      'standard-whats-new-2' => File.join(source_metadata_folder, 'standard_whats_new_2.txt'),
-      'standard-whats-new-3' => File.join(source_metadata_folder, 'standard_whats_new_3.txt'),
-      'standard-whats-new-4' => File.join(source_metadata_folder, 'standard_whats_new_4.txt'),
-      'app_store_screenshot-1' => File.join(source_metadata_folder, 'promo_screenshot_1.txt'),
-      'app_store_screenshot-2' => File.join(source_metadata_folder, 'promo_screenshot_2.txt'),
-      'app_store_screenshot-3' => File.join(source_metadata_folder, 'promo_screenshot_3.txt'),
-      'app_store_screenshot-4' => File.join(source_metadata_folder, 'promo_screenshot_4.txt'),
-      'app_store_screenshot-5' => File.join(source_metadata_folder, 'promo_screenshot_5.txt'),
-      'app_store_screenshot-6' => File.join(source_metadata_folder, 'promo_screenshot_6.txt'),
-      'app_store_screenshot-7' => File.join(source_metadata_folder, 'promo_screenshot_7.txt')
+      'standard-whats-new-1' => File.join(custom_metadata_folder, 'standard_whats_new_1.txt'),
+      'standard-whats-new-2' => File.join(custom_metadata_folder, 'standard_whats_new_2.txt'),
+      'standard-whats-new-3' => File.join(custom_metadata_folder, 'standard_whats_new_3.txt'),
+      'standard-whats-new-4' => File.join(custom_metadata_folder, 'standard_whats_new_4.txt'),
+      'app_store_screenshot-1' => File.join(custom_metadata_folder, 'promo_screenshot_1.txt'),
+      'app_store_screenshot-2' => File.join(custom_metadata_folder, 'promo_screenshot_2.txt'),
+      'app_store_screenshot-3' => File.join(custom_metadata_folder, 'promo_screenshot_3.txt'),
+      'app_store_screenshot-4' => File.join(custom_metadata_folder, 'promo_screenshot_4.txt'),
+      'app_store_screenshot-5' => File.join(custom_metadata_folder, 'promo_screenshot_5.txt'),
+      'app_store_screenshot-6' => File.join(custom_metadata_folder, 'promo_screenshot_6.txt'),
+      'app_store_screenshot-7' => File.join(custom_metadata_folder, 'promo_screenshot_7.txt')
```

Comment:
These strings are the copy for the App Store screenshots and fallback for release notes with no translation–currently unused.

Historically, these have lived in `fastlane/appstoreres/metadata/source/` and we forgot to account for
that when splitting up the `Fastfile` and updating the localization tooling.

I don't know the consequences of adding custom `.txt` sources to the folders `deliver` looks into. Given this work is happening in `release/19.8` branch context, I'd rather do as little changes as necessary, and save bigger restructuring for a PR against `trunk` on which we can discuss and iterate without the release deadline looming
upon us.

---

The fact that we have the machine generated commits in this diff and that the values read from these sources in the `.po` haven't changed is proof this update works correctly.